scipy/sparse/_base.py

```diff
@@ -1348,6 +1381,11 @@ def getnnz(self, axis=None):
         See also
         --------
         count_nonzero : Number of non-zero entries
+
+        .. deprecated:: 1.11.0
+           This method will be removed in SciPy 1.13.0. Use `X.nnz`
+           instead.  The `axis` argument will no longer be supported;
+           please let us know if you still need this functionality.
```

Comment:
This is breaking because it's in the `See also` section. I would move it before parameters (GH does not allow me to make a suggestion there sorry)